.github/workflows/test_bfrescox_sdist.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         # Python versions should be maintained consistent with those in setup.py
-        os: [ubuntu-22.04, ubuntu-24.04, macos-13, macos-14, macos-15]
+        os: [ubuntu-22.04, ubuntu-24.04, macos-14, macos-15]
```

Comment:
With this setup, we are no longer testing on Intel-based Macs.  It looks like `macos-15-intel` was added to address this when macOS 13 was deprecated.

Please see if tests pass on that new runner.  Ditto for `pro` package.